prefect_databricks/flows.py

```diff
@@ -314,6 +325,24 @@ async def jobs_runs_submit_and_wait_for_completion_flow(notebook_path, **base_pa
                 jobs_runs_life_cycle_state.lower(),
                 poll_frequency_seconds,
             )
+            for runs_task in jobs_runs_tasks:
+                if "run_id" not in runs_task or "state" not in runs_task:
+                    continue
+                runs_task_id = runs_task["run_id"]
+                runs_task_run_page_url = ""
+                if "run_page_url" in runs_task:
+                    runs_task_run_page_url = runs_task["run_page_url"]
+                runs_task_state = runs_task["state"]
+                runs_task_life_cycle_state = runs_task_state["life_cycle_state"]
+                runs_task_state_message = runs_task_state["state_message"]
+
+                logger.info(
+                    "Databricks Jobs Runs Task (ID %s) has '%s', '%s'. %s",
+                    runs_task_id,
+                    runs_task_life_cycle_state.lower(),
+                    runs_task_state_message,
+                    runs_task_run_page_url,
```

Comment:
Would it be better to display the task run page URLs above (next to the parent URL) so it doesn't crowd the logger?

Reply:
Yeah, possibly so, I'll review this in the AM.

Reply:
Added